client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -302,17 +322,27 @@ ShippingLabels.propTypes = {
 	setValue: PropTypes.func.isRequired,
 };
 
-export default connect( ( state, { siteId } ) => {
-	return {
-		isLoading: areSettingsFetching( state, siteId ),
-		pristine: isPristine( state, siteId ),
-		paymentMethods: getPaymentMethods( state, siteId ),
-		selectedPaymentMethod: getSelectedPaymentMethodId( state, siteId ),
-		paperSize: getPaperSize( state, siteId ),
-		storeOptions: getLabelSettingsStoreOptions( state, siteId ),
-		canEditPayments: userCanManagePayments( state, siteId ),
-		canEditSettings: userCanManagePayments( state, siteId ) || userCanEditSettings( state, siteId ),
-		emailReceipts: getEmailReceipts( state, siteId ),
-		...getMasterUserInfo( state, siteId ),
-	};
-} )( localize( ShippingLabels ) );
+export default connect(
+	( state, { siteId } ) => {
+		return {
+			isLoading: areSettingsFetching( state, siteId ),
+			pristine: isPristine( state, siteId ),
+			paymentMethods: getPaymentMethods( state, siteId ),
+			selectedPaymentMethod: getSelectedPaymentMethodId( state, siteId ),
+			paperSize: getPaperSize( state, siteId ),
+			storeOptions: getLabelSettingsStoreOptions( state, siteId ),
+			canEditPayments: userCanManagePayments( state, siteId ),
+			canEditSettings:
+				userCanManagePayments( state, siteId ) || userCanEditSettings( state, siteId ),
+			emailReceipts: getEmailReceipts( state, siteId ),
+			...getMasterUserInfo( state, siteId ),
+		};
+	},
+	dispatch =>
+		bindActionCreators(
```

Comment:
Missing import `bindActionCreators`, the settings page isn't loading at all for me.

Reply:
Whoops — should work now.